### DISCUSSION TITLE
Mobile features testing

### DISCUSSION THREAD

**Discussion by tripleo1** [opuntiaOS-Project/opuntiaOS#48] [General]
How do you test the mobile features?

Is there specific hardware that you use?

How do you build it?

**Comment by nimelehin** [comment]
Mobile features are tested in QEMU.

If we talk about ARM, currently we have support only for Cortex-A15 Core. During these development stages the main target is QEMU's vexpress-a15, so the set of implemented drivers is limited to run on vexpress-a15.

To built it use `./gn_gen.sh --device_type mobile` than `cd out` and `./all.sh` [learn more](https://github.com/opuntiaOS-Project/opuntiaOS/blob/master/docs/build.md)
